src/Resources/config/app/config.yml

```diff
@@ -160,6 +161,36 @@ sylius_ui:
                 credit_memos:
                     template: '@SyliusRefundPlugin/Order/Shop/creditMemos.html.twig'
                     priority: 5
+        sylius_refund.shop.order.credit_memos.list.actions.buttons:
+            blocks:
+                download:
+                    template: '@SyliusRefundPlugin/Order/Shop/_downloadButton.html.twig'
+                    enabled: '%sylius_refund.pdf_generator.enabled%'
+                    priority: 50
+
+        sylius_refund.admin.order.credit_memos.list.actions.buttons:
+            blocks:
+                details:
+                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_detailsButton.html.twig'
+                    priority: 50
+                download:
+                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_downloadButton.html.twig'
+                    enabled: '%sylius_refund.pdf_generator.enabled%'
+                    priority: 40
+        sylius_refund.admin.order.credit_memo.left_metadata:
+            blocks:
+                header:
+                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_header.html.twig'
+                    priority: 50
+                breadcrumb:
+                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_breadcrumb.html.twig'
+                    priority: 40
+        sylius_refund.admin.order.credit_memo.right_metadata:
+            blocks:
+                download:
+                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_downloadButton.html.twig'
+                    enabled: '%sylius_refund.pdf_generator.enabled%'
+                    priority: 50
```

Comment:
Why did you start with the priority 50 for new blocks in new events?
```suggestion

        sylius_refund.shop.order.credit_memos.list.actions.buttons:
            blocks:
                download:
                    template: '@SyliusRefundPlugin/Order/Shop/_downloadButton.html.twig'
                    enabled: '%sylius_refund.pdf_generator.enabled%'
                    priority: 10

        sylius_refund.admin.order.credit_memos.list.actions.buttons:
            blocks:
                details:
                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_detailsButton.html.twig'
                    priority: 20
                download:
                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_downloadButton.html.twig'
                    enabled: '%sylius_refund.pdf_generator.enabled%'
                    priority: 10

        sylius_refund.admin.order.credit_memo.left_metadata:
            blocks:
                header:
                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_header.html.twig'
                    priority: 20
                breadcrumb:
                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_breadcrumb.html.twig'
                    priority: 10

        sylius_refund.admin.order.credit_memo.right_metadata:
            blocks:
                download:
                    template: '@SyliusRefundPlugin/Order/Admin/CreditMemo/_downloadButton.html.twig'
                    enabled: '%sylius_refund.pdf_generator.enabled%'
                    priority: 10
```

Reply:
I thought it would be nice to be able to add future blocks with a lower priority, starting from 50 gives, for instance, the possibility to add 4 new buttons after the download button. What do you think about it?

Reply:
But we will still be able to add blocks in future, I would be for starting from `10` as we did on Sylius to be consistent with our approach there